sdk/servicebus/service-bus/src/models.ts

```diff
@@ -4,6 +4,46 @@
 import { OperationOptionsBase } from "./modelsToBeSharedWithEventHubs";
 import Long from "long";
 
+/**
+ * Context for the error given in processError.
+ */
+export interface ProcessErrorContext {
+  /**
+   * The operation where the error originated.
+   */
+  errorSource: /**
+   * Covers errors that occur when the user (or autoComplete) completes a message.
```

Comment:
user?

Reply:
Removed the 'user' part and also explained a bit more in the text. Let me know if you like it better.